dockerfiles/java_installer.sh

```
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cd /mnt/function
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)